browser/scripts/abstractPlugins/abstractObjectGazePlugin.js

```diff
@@ -9,7 +9,7 @@
 				return
 
 			subgraph.nodes.map(function(node) {
-				if (node.plugin.id === 'graph')
+				if (E2.GRAPH_NODES.indexOf(node.plugin.id) > -1)
```

Comment:
@gmarinov it's better than checking for 'graph' ;) which broke entity, entity_component and loop ...